bus/event_bus.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bus
+
+import asaskevichEventBus "github.com/asaskevich/EventBus"
+
+// EventBus allows subscribing and publishing data by topic
+type EventBus interface {
```

Comment:
still not a fan of central declarations of interface

Reply:
That's why other components, who needs it, don't import this interface, but declare interface of their own - it's just the reference of what `EventBus` we use when we create it, which is much simpler than the full implementation of asaskevich event bus.